CHANGES.md

```diff
@@ -1,5 +1,38 @@
 # Changelog
 
+# 0.14.6
+
+### New
+
+* IO manager for materializing assets to GCS. Specify the GCS asset IO manager using the following config for `resource_defs` in `AssetGroup`:
```

Comment:
Maybe to keep consistent with the other items, consider adding a verb?  (e.g. `Added a new IO manager...  You can specify the GCS asset IO manager by using ...`)